src/commands/run.js

```diff
@@ -69,6 +69,12 @@ exports.builder = function (yargs) {
   }).option('build-script', {
     description: 'The npm script that will be run when building the app',
     default: 'build',
+  }).option('http', {
+    description: 'URL for where your app is served from e.g. localhost:1234',
+    default: null,
+  }).option('http-served-from', {
```

Comment:
Changed the flag `--served-at` to `--http-served-from` so it is more descriptive to what the flag does.